converter/src/main/java/gov/cms/qpp/conversion/model/validation/SupplementalData.java

```diff
@@ -0,0 +1,117 @@
+package gov.cms.qpp.conversion.model.validation;
+
+import gov.cms.qpp.conversion.model.TemplateId;
+import java.util.Arrays;
+import java.util.EnumMap;
+import java.util.Map;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.EnumSet'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)